engine/src/main/java/org/pentaho/di/trans/steps/sqlfileoutput/SQLFileOutputMeta.java

```diff
@@ -602,13 +602,13 @@ public void check( List<CheckResultInterface> remarks, TransMeta transMeta, Step
         if ( !Utils.isEmpty( tablename ) ) {
           String schemaTable = databaseMeta.getQuotedSchemaTableCombination( schemaName, tablename );
```

Comment:
Think we can remove this line or only define in case of error

Reply:
This one is used at least 2 times, if not 3. I'll leave this one here.